tests/e2e/conftest.py

```diff
@@ -95,3 +95,19 @@ def random_str(n: int) -> str:
             "authorName": random_str(1536),
         }
         driver.execute_query(insert_query, parameters)
+
+
+@pytest.fixture(scope="module")
+def setup_schema(driver):
```

Comment:
Naming suggestions are welcome.

Reply:
How about `setup_neo4j_for_schema_query`? We can also rename the other `setup_neo4j` fixture to `setup_neo4j_for_retrieval`

Reply:
Sounds good! 👍

Reply:
[8eacfb6](https://github.com/neo4j/neo4j-genai-python/pull/36/commits/8eacfb6020596ab2cbb58eb1c67bfa40894b45de)